PSPs/drafts/psp-17.md

```diff
@@ -91,13 +95,13 @@ struct Approval {
 ```
 
 ### Errors
-Suggested methods don't return `Result`. Instead, they panic.
+Suggested methods don't return `Result` (except `on_received`). Instead, they panic.
 This panic can contain one of the following messages:
 
 ```rust
-pub enum PSP20Error {
+pub enum PSP17Error {
  /// Unknown error type for cases if writer of traits added own restrictions
- Unknown(&'static str),
+ Unknown(String),
```

Comment:
Let's maybe call it Custom. It substrate they will do the same for dymanic errors: https://github.com/paritytech/substrate/issues/5239#issuecomment-858451861